content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -92,39 +92,71 @@ If you are using Heroku, follow the steps below.
 4. Click **View Credentials**.
 5. From the **URI** field, copy the entire connection string.
 
-**Checkpoint:** The Heroku connection string has the following format:
+**Checkpoint:** The database connection string has the following format:
 
 ```
-postgres://__USER__:__PASSWORD__@__HOST__/__DATABASE__
+__CONNECTOR__://__USER__:__PASSWORD__@__HOST__/__DATABASE__
```

Comment:
`postgresql://` is fine here as we tell them to create a PostgreSQL database.

Reply:
Ok, restored it to postgresql://